docs/includes/openapi.adoc

```diff
@@ -18,6 +18,155 @@
+or `META-INF/openapi.json`. Tools such as Swagger let you describe your app's API
+and they then generate an OpenAPI document file which you can include in your application
+so OpenAPI can use it.
+
+===== Write and configure a model reader class
+Write a Java class that implements the OpenAPI
+link:{microprofile-open-api-javadoc-url}/OASModelReader.html[`org.eclipse.microprofile.openapi.OASModelReader`] interface. Your
+model reader code programmatically adds elements to the internal model that OpenAPI
+builds.
+
+Change your application's MP configuration to set `mp.openapi.model.reader` as the
+fully-qualified class name of your class.
+
+===== Write and configure a filter class
+Write a Java class that implements the OpenAPI
+link:{microprofile-open-api-javadoc-url}/OASFilter.html[`org.eclipse.microprofile.openapi.OASFilter`] interface.
+As OpenAPI composes its internal model, it invokes your filter with each
+model element _before_ adding the element to the model. Your filter can
+accept the element as-is, modify it, or suppress it.
+
+Change your application's configuration to set `mp.openapi.filter` as the full-qualified
+class name of your class.
+
+// end::furnish-openapi-info[]
+
+// tag::usage-access-endpoint[]
+=== Accessing the REST Endpoint
+Once you add the {flavor-uc} OpenAPI dependency to your
+ifdef::mp-flavor[project,]
+ifdef::se-flavor[project and add code to create the `OpenAPISupport` object to your routing,]
+your application will automatically respond to the built-in endpoint --
+`/openapi` -- and it will return the OpenAPI document describing the endpoints
+in your application.
+
+By default, per the MicroProfile OpenAPI spec, the default format of the OpenAPI document is YAML.
+There is not yet an adopted IANA YAML media type, but a proposed one specifically
+for OpenAPI documents that has some support is `application/vnd.oai.openapi`.
+That is what Helidon returns, by default.
+
+In addition, a client can specify the HTTP header `Accept:` as either `application/vnd.oai.openapi+json` or
+`application/json` to request JSON. Alternatively, the client can pass the query parameter `format` as either `JSON`
+or `YAML` to receive `application/json` or `application/vnd.oai.openapi` (YAML) output, respectively.
+// end::usage-access-endpoint[]
+
+// tag::api[]
+ifdef::mp-flavor[]
+The link:{microprofile-open-api-spec-url}[MicroProfile OpenAPI specification] gives a listing and brief examples of the annotations you can add to your code to convey OpenAPI information.
+endif::[]
+
+The link:{microprofile-open-api-javadoc-base-url}[MicroProfile OpenAPI JavaDocs] give full details of the
+ifdef::mp-flavor[annotations and the other]
+classes and interfaces you can use in your code.
+ifdef::se-flavor[]
+Remember that, although the JavaDocs describe annotations, Helidon {flavor-uc} does not support them.
+endif::[]
+
+// end::api[]
+
 // tag::common-config[]
 Helidon {h1-prefix} also supports additional properties specific to Helidon.
```

Comment:
Drop one "Helidon"?

Reply:
That tag (which I never closed anyway) is never used. Removed.